volatility3/framework/plugins/windows/processghosting.py

```diff
@@ -89,7 +202,9 @@ def run(self):
                 ("PID", int),
                 ("Process", str),
                 ("FILE_OBJECT", format_hints.Hex),
-                ("DeletePending", str),
+                ("DeletePending", int),
+                ("DeleteOnClose", int),
+                ("Base", format_hints.Hex),
```

Comment:
Is this the ordering we want?  Feels like Base should be earlier (next to Process maybe?)  Entirely your call, just looks weird after two small int values?

Reply:
I will change it to be more like the other plugins.

Reply:
Done in latest commit.